doc/api/readline.md

```diff
@@ -628,12 +628,12 @@ the best compatibility if it defines an `output.columns` property and emits
 a `'resize'` event on the `output` if or when the columns ever change
 ([`process.stdout`][] does this automatically when it is a TTY).
 
-When creating a `readline.Interface` using `stdin` as input, the program
+When creating a `readline.Interface` using `stdin` as input stream, the program
 will not terminate until it receives `EOF` (<kbd>Ctrl</kbd>+<kbd>D</kbd> on
-Linux/MacOS, <kbd>Ctrl</kbd>+<kbd>Z</kbd> followed by <kbd>return</kbd> on
+Linux/macOS, <kbd>Ctrl</kbd>+<kbd>Z</kbd> followed by <kbd>Return</kbd> on
 Windows).
 If you want your application to exit without waiting for user input, you can
-`unref` the standard input:
+`unref` the standard input stream:
```

Comment:
Thinking about this a bit more, it's probably worth adding a link for folks who want to dig deeper:
```suggestion
[`unref`][] the standard input stream:
```

And add a reference at the end of the file:

```markdown
[`unref`]: net.md#net_socket_unref
```